lib/identity/data/data_sources/smt_data_source.dart

```diff
@@ -255,6 +255,7 @@ class SMTDataSource {
     final root = await _storageSMTDataSource.getRoot(
         storeName: storeName, did: did, privateKey: privateKey);
     var nextKey = root;
+    // ignore: dead_code
```

Comment:
I fixed this dead code, was because the use of continue instead of break, dont ignore it